package.json

```diff
@@ -658,6 +683,11 @@
                 "title": "Create",
                 "category": "Kubernetes"
             },
+            {
```

Comment:
I don't see an entry in the `menus.commandPalette` section, so these commands will be invokable from the command palette. If that's intentional, then you need to handle the case where they are invoked without a URI (i.e. the argument needs to be optional).

Reply:
What do I need to do to hide them from the command pallette? (thanks)